docroot/themes/custom/dcp/dcp.libraries.yml

```diff
@@ -3,7 +3,7 @@ global-styling:
   css:
     base:
       css/style.css: {}
-      single-page/style.css: {}
+#      single-page/style.css: {}
```

Comment:
any reason? why are we commenting it for now!! this css is being used for single page site

Reply:
@prashant114606 this is conflicting with the top style.css file

Reply:
this needs a discussion!! let's discuss this in next call
cc @ajitdev @dipakmdhrm